isis/src/control/objs/BundleUtilities/BundleObservation.cpp

```diff
@@ -1411,4 +1363,231 @@ QString BundleObservation::formatBundleOutputString(bool errorPropagation, bool
 
     return finalqStr;
   }
+
+
+  bool BundleObservation::computeTargetPartials(matrix<double> &coeffTarget, BundleMeasure &measure,
+                                                BundleSettingsQsp &bundleSettings, BundleTargetBodyQsp &bundleTargetBody) {
+    if (bundleSettings->solveTargetBody()) {
```

Comment:
It doesn't make sense to check if we're solving for target body in the function specifically for solving for target body. Assume this is true already by here.

Reply:
Yeah, I agree.